src/Tests/Cache/PhpRedisUnitTest.php

```diff
@@ -45,7 +44,11 @@ public function containerBuild(ContainerBuilder $container) {
    *   A new PhpRedis cache backend.
    */
   protected function createCacheBackend($bin) {
-    $cache = \Drupal::service('cache.backend.redis')->get($bin);
+    $cache = new PhpRedis(
```

Comment:
Well, the downside is that we then don't test the backend ;) Not sure I fully understand the problem...